src/server/auth/testing/auth.go

```diff
@@ -13,91 +13,101 @@ import (
 type InactiveAPIServer struct{}
 
 // Activate implements the Activate RPC, but just returns NotActivatedError
-func (a *InactiveAPIServer) Activate(ctx context.Context, req *auth.ActivateRequest) (resp *auth.ActivateResponse, retErr error) {
+func (a *InactiveAPIServer) Activate(_ context.Context, _ *auth.ActivateRequest) (*auth.ActivateResponse, error) {
```

Comment:
You can actually just remove the argument names entirely here and write this is:

```
func (a *InactiveAPIServer) Activate(context.Context, *auth.ActivateRequest) (*auth.ActivateResponse, error) {
```
saves us another 4 characters :)